src/components/cms/BotBuilder/BotBuilder.js

```diff
@@ -228,7 +228,7 @@ class BotBuilder extends React.Component {
           >
             <Link
               to={
-                '/botbuilder/botwizard?name=' +
+                '/mybots/botwizard?name=' +
```

Comment:
Don't change that, please.

That was not the idea. Actually "botwizard" itself should be renamed to "/botbuilder". However, I did not mention this as it could be confusing in this issue.

Reply:
@mariobehling  So should I rename it to /botbuilder or leave it like earlier.

Reply:
take out "mybots" in the botwizard URL